workflow-templates/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: Link Markdown
+
+on: [push, pull_request]
+
+jobs:
+  check_links:
+    name: Links
+    runs-on: ubuntu-latest
+    steps:
+      - name: Actions Checkout
+        uses: actions/checkout@v2
+        with: 
+          token: '${{ secrets.GH_PAT }}'
+          persist-credentials: false
+      - name: Check Links
+        uses: gaurav-nelson/github-action-markdown-link-check@v1
+        with: 
+          config-file: '.github/workflows/mlc_config.json'
+          folder-path: "."
+
+  lint-check:
+    name: Lint Markdown Check
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check Out Code
+      uses: actions/checkout@v2
+    - name: Lint Changelog Files
+      uses: avto-dev/markdown-lint@v1
+      with:
+        rules: '/lint/rules/changelog.js'
+        config: '/lint/config/changelog.yml'
```

Comment:
Where are these files located?

Reply:
Those files are in the repo where we are calling the tool.
"https://github.com/avto-dev/markdown-lint"